examples/atomsbase.jl

```diff
@@ -5,6 +5,49 @@
 # structures to run a calculation as is demonstrated here.
 
 using DFTK
+
+# ## Feeding an AtomsBase AbstractSystem to DFTK
+# In this example we construct a silicon system using the `ase.build.bulk` routine
+# from the [atomistic simulation environment](https://wiki.fysik.dtu.dk/ase/index.html)
+# (ASE), which is exposed by [ASEconvert](https://github.com/mfherbst/ASEconvert.jl)
+# as an AtomsBase `AbstractSystem`.
+
+## Construct bulk system and convert to an AbstractSystem
+using ASEconvert
+system_ase = ase.build.bulk("Si")
+system = pyconvert(AbstractSystem, system_ase)
+
+# To use an AbstractSystem in DFTK, we attach pseudopotentials, construct a DFT model,
+# discretise and solve:
+system = attach_psp(system; Si="hgh/lda/si-q4")
+
+model  = model_LDA(system; temperature=1e-3)
+basis  = PlaneWaveBasis(model; Ecut=15, kgrid=[4, 4, 4])
+scfres = self_consistent_field(basis, tol=1e-8);
+
+# If we did not want to use ASE we could of course use any other package,
```

Comment:
Using commas like this ("any other package, which yields an AS object, ...") is a germanism: remove the first comma